binplace.targets

```diff
@@ -31,32 +31,24 @@
   </Target>
 
   <Target Name="GetTargetingPackDir">
-    <ItemGroup Condition="'$(NuGetTargetMoniker)' == '' Or $(NuGetTargetMoniker.Contains('NETCoreApp')) Or $(NuGetTargetMoniker.Contains('NETStandard'))">
-      <TargetingPackDir Include="$(BinDir)netcoreapp\TargetingPack" />
+    <ItemGroup>
+      <TargetingPackDir Include="$(BinDir)ref\$(BuildConfiguration)" />
```

Comment:
Why do we still need to calculate this properties inside a Target? can't we just pomote them to a static property?